Makefile

```diff
@@ -107,8 +107,13 @@ manifests: __controller-gen ## Generates k8s yaml for eraser deployment.
 		k8s.gcr.io/kustomize/kustomize:v${KUSTOMIZE_VERSION} build \
 		--load_restrictor LoadRestrictionsNone /eraser/third_party/open-policy-agent/gatekeeper/helmify | go run third_party/open-policy-agent/gatekeeper/helmify/*.go
 
-generate: __controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+generate: __conversion-gen __controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(CONVERSION_GEN) \
+		--output-base=/eraser \
+		--input-dirs=./api/v1,./api/v1alpha1,./api/unversioned \
```

Comment:
```suggestion
		--input-dirs=./api/... \
```

Reply:
done

Reply:
Not sure why, but when I do this I get the error
```
docker run --rm -v /opt/src/go/eraser:/eraser eraser-tooling conversion-gen \
        --output-base=/eraser \
        --input-dirs=./api/... \
        --go-header-file=./hack/boilerplate.go.txt \
        --output-file-base=zz_generated.conversion
W1216 16:01:48.513028       1 parse.go:247] Ignoring directory ./api: No files for pkg "./api"

```

Reply:
i think you need an empty package for this to work like https://github.com/open-policy-agent/frameworks/blob/master/constraint/pkg/apis/externaldata/group.go

Reply:
done